fabric_cf/actor/core/core/broker_policy.py

```diff
@@ -119,12 +121,16 @@ def get_client_id(self, *, reservation: ABCServerReservation) -> ID:
         return reservation.get_client_auth_token().get_guid()
 
     @staticmethod
-    def get_broker_query_model_query(*, level: int) -> dict:
+    def get_broker_query_model_query(*, level: int, bqm_format: int = GraphFormat.GRAPHML.value) -> dict:
```

Comment:
Are you sure you need .value and can't just use GraphFormat.GRAPHML?

Reply:
Updated the fix